plans/smoke.fmf

```diff
@@ -0,0 +1,9 @@
+summary: Run smoke test
+discover:
```

Comment:
I believe `discover` steps does not need to be here. You are running a script in `execute` a script, thus discovered tests will be ignored I believe

Reply:
@psss right?

Reply:
Thanks for your review! Solved